packages/itwinui-css/src/surface/surface.scss

```diff
@@ -18,9 +18,37 @@
   border-radius: var(--iui-surface-border-radius);
   box-shadow: var(--iui-surface-elevation);
   color: var(--iui-color-text);
+
+  &:where([data-iui-layout='true']) {
+    display: flex;
+    flex-direction: column;
+  }
+}
+
+$surface-padding: var(--iui-size-2xs);
+
+@mixin iui-surface-header {
+  display: flex;
+  flex-shrink: 0;
+  align-items: center;
+  overflow: hidden;
+  border-bottom: 1px solid var(--iui-surface-border-color);
+  padding: $surface-padding;
+  scrollbar-gutter: stable both-edges;
 }
 
-// Public facing surface component.
-@mixin iui-surface-component {
-  @include iui-surface;
+@mixin iui-surface-body {
+  display: flex;
```

Comment:
does this need to be flex?

Reply:
It is not, removed.

Reply:
Oh wait, it might be needed.  I was using this to center the empty state.  Here's how it looks after removing `display: flex`:

![Screenshot 2023-02-23 at 11 08 33 AM](https://user-images.githubusercontent.com/849817/220964036-ccaf7e40-8b58-4c6d-9a3c-2184452f1b82.png)

Reply:
well, centering the content should be left up to the user

Reply:
What about leaving `display: flex` or adding `position: relative` so the user can easily achieve vertical centering?

Reply:
my concern is that `display: flex` can be pretty invasive, and could unnecessarily break styles. same with `positive: relative`, it can create an undesirable containing block.

plus, the user can always easily add custom styles and we even give them a `<Flex>` utility, so i dont think we should do it for them.

Reply:
Ok, I'll leave `display: flex` out of the CSS & will add an inline style to apply `display: flex` just to keep that 1 empty state demo working correctly.